.lfsconfig

```diff
@@ -0,0 +1,5 @@
+[lfs]
+# codeql is publicly forked by many users, and we don't want any LFS file polluting their working
+# copies. We therefore exclude everything by default.
+# For files required by bazel builds, use rules in `misc/bazel/lfs.bzl` to download them on demand.
+fetchinclude = /nothing
```

Comment:
yeah, agreed, will do!

Reply:
ah, I remembered why I had gone for `fetchinclude` rather than `fetchexclude`: the latter has precedence over the former, so if one wants to override these settings in order to fetch some specific files it becomes non trivial.

Say we start having more of such lazy LFS files (I'm planning to use this for Swift as well), and you want to fetch the kotlin files. With `fetchinclude=/nothing` in `.lfsconfig`, it suffices for you to `git config lfs.fetchinclude=/java/kotlin-extractor/**`, which will override the `/nothing` and give you what you want, withoug excluding other files. 

With `fetchexclude=*` in `.lfsconfig`, you would need to (say) do something like `git config lfs.fetchexclude=swift/**`, or exclude all directories save for `/java` just to be forward compatible with the exclusion rules...